manubot/cite/csl_item.py

```diff
@@ -152,6 +152,32 @@ def clean(self, prune: bool = True):
             self.validate_against_schema()
         return self
 
+    def set_date_variable(self, iso_date, variable="issued"):
+        """
+        iso_date: date either as a string (in the form YYYY, YYYY-MM, or YYYY-MM-DD)
+            or as a Python date object (datetime.date or datetime.datetime).
+        variable: which variable to assing the date to.
+        """
+        date_parts = date_to_date_parts(iso_date)
+        if date_parts:
+            self[variable] = {"date-parts": [date_parts]}
+        return self
+
+    def get_date_variable(self, variable="issued", fill=False):
```

Comment:
This behavior is probably too complex to consider the date as a property, right?

Reply:
With the `variable` and `fill` options, I think yes... cannot fit within a property.